src/app/modules/item/components/chapter-children/chapter-children.component.ts

```diff
@@ -23,7 +25,7 @@ export class ChapterChildrenComponent implements OnChanges {
   private reloadData() {
     if (this.itemData.attemptId) {
       this.state = 'loading';
-      this.getItemChildrenService
```

Comment:
The goal of the subscription is to cancel it when reloading a new item... so you need first to cancel the subscription if there is one (to cancel the request).

Reply:
fixed